tests/bluetooth/bsim_bt/bsim_test_app/src/test_connect2.c

```diff
@@ -0,0 +1,172 @@
+/*
+ * Copyright (c) 2015-2016 Intel Corporation
+ * Copyright (c) 2017-2019 Oticon A/S
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+#include "kernel.h"
+
+#include "bs_types.h"
+#include "bs_tracing.h"
+#include "time_machine.h"
+#include "bstests.h"
+
+#include <zephyr/types.h>
+#include <stddef.h>
+#include <errno.h>
+#include <zephyr.h>
+#include <misc/printk.h>
+
+#include <bluetooth/bluetooth.h>
+#include <bluetooth/hci.h>
+#include <bluetooth/conn.h>
+#include <bluetooth/uuid.h>
+#include <bluetooth/gatt.h>
+#include <misc/byteorder.h>
+
+#include <gatt/hrs.h>
+#include <gatt/bas.h>
+
+static struct bt_conn *default_conn;
+
+/*
+ * Basic connection test:
+ *   We expect a central to connect to us.
+ *
+ *   The thread code is mostly a copy of the peripheral_hr sample device
+ */
+
+#define WAIT_TIME 5 /*seconds*/
+extern enum bst_result_t bst_result;
+
+#define FAIL(...)					\
```

Comment:
How come you're redefining these here? Wouldn't it make sense to have them in some local header file?